server/helpers/identification.js

```diff
@@ -0,0 +1,14 @@
+import { User } from '../database/models';
+import responseMessage from './responseMessage';
+
+const getUserId = async (param, response) => {
+  try {
+    const field = (/\D/g.test(param)) ? { email: param } : { id: param };
+    const user = await User.findOne({ where: field });
```

Comment:
What happens when this doesn't find a user?

Reply:
it returns undefined then whoever wants to use the function will check for undefined.